src/main.cs

```diff
@@ -119,8 +119,10 @@ public class AboutProgramm : IEplAction
 
         public bool Execute(ActionCallingContext ctx)
         {
-            MessageBox.Show($"Версия надстройки - {GetVersion()}", 
-                "Версия надстройки", MessageBoxButtons.OK, 
+            string messageLicense = $"Версия надстройки -{GetVersion()}\n" +
```

Comment:
```suggestion
            string message = $"Версия надстройки - {GetVersion()}\n" +
```

Отступ между версией и тире надо добавить. И переименовать переменную в message